pkgs/by-name/fl/flatter/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, unstableGitUpdater
+, cmake
+, blas
+, gmp
+, mpfr
+, fplll
+, eigen
+}:
+
+stdenv.mkDerivation {
+  pname = "flatter";
+  version = "unstable-2023-08-10";
+
+  src = fetchFromGitHub {
+    owner = "keeganryan";
+    repo = "flatter";
+    rev = "500e31df6b7308e8101b2a4a9cc816bf8f483417";
+    hash = "sha256-STYx7cXvkcF+KqrG32pN16HWfEScc0zxkmOmfv43zIw=";
+  };
+
+  passthru.updateScript = unstableGitUpdater {};
+
+  nativeBuildInputs = [ cmake blas gmp mpfr fplll eigen ];
```

Comment:
Does that work with `strictDeps = true`?

Reply:
With `strictDeps = true`, building fails as CMake fails to find the needed libraries (BLAS, GMP, etc.). `strictDeps = true` builds fine if these are moved into `buildInputs` instead. Is this preferrable?

Reply:
Yes, libraries [usually go to `buildInputs`](https://nixos.org/manual/nixpkgs/stable/#ssec-stdenv-dependencies-overview).